bin/internal/shared.sh

```diff
@@ -41,8 +41,14 @@ function _rmlock () {
 
 # Determines which lock method to use, based on what is available on the system.
 # Returns a non-zero value if the lock was not acquired, zero if acquired.
+#
+# There is macOS version of flock but it's not compatible: 1) it doesn't support 
+# `exclusive` option and 2) it doesn't support `--` option format. We can't use
```

Comment:
Wordsmithing a bit, avoid repeating and usage of "we"


```
...We can't use
# flock even if it's installed on macOS, otherwise this function will fail and 
# flutter command will be blocked by it.
```
change to something like?
```
Do not use flock on macOS, otherwise this function will fail and the script will stall.
```

Reply:
Sounds good! Thanks for the suggestion!